chromadb/test/test_logservice.py

```diff
@@ -0,0 +1,147 @@
+# type: ignore
```

Comment:
I think we should plug this into test_producer_consumer if that makes sense (I would have to look at the test again to be sure)

Reply:
that test doesn't go through front end apis and main logic is testing topic assignments and seqIds. Since we don't need that for logservice, I create a new test. We can clean up the interface later when we have time.